testing/test_parseopt.py

```diff
@@ -288,8 +288,16 @@ def test_multiple_metavar_help(self, parser: parseopt.Parser) -> None:
 
 
 def test_argcomplete(testdir, monkeypatch) -> None:
-    if not shutil.which("bash"):
-        pytest.skip("bash not available")
+    try:
+        bash_version = subprocess.run(
+            ["bash", "--version"], capture_output=True, universal_newlines=True
```

Comment:
`capture_output` was added in Python 3.7, so I suspect this will fail with Python 3.5 and 3.6 (but hidden by the overly broad catching below).

Reply:
Missed this, will fix.